tests/Embedded/SourceTest.php

```
@@ -0,0 +1,334 @@
+    {
+        self::assertSame('<source dir="ltr">', Source::tag()->dir('ltr')->render());
+    }
+
+    public function testRenderWithDirUsingEnum(): void
+    {
+        self::assertSame('<source dir="ltr">', Source::tag()->dir(Direction::LTR)->render());
+    }
+
+    public function testRenderWithGlobalDefaultsAreApplied(): void
+    {
+        SimpleFactory::setDefaults(Source::class, ['class' => 'default-class']);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\UIAwesome\Html\Core\Factory\SimpleFactory' in method 'testRenderWithGlobalDefaultsAreApplied'. (undefined)

(StaticAccess)